apps/starter-nextjs-trpc-clerk/utils/xata.ts

```diff
@@ -0,0 +1,61 @@
+// Generated by Xata Codegen 0.21.0. Please do not edit.
```

Comment:
won't committing this make whoever runs the starter to try to connect to _our_ db without having access?
Maybe this file shouldn't be committed. Wdyt?

Reply:
I can of like having this file as reference, so the project is not broken when you clone it. But I will remove the defaultUrl bellow, I think this could be a nice tradeoff.